frontend/src/app/signUp/IdentityVerification/types.d.ts

```diff
@@ -7,7 +7,13 @@ interface Question {
 }
 
 interface Answers {
-  [outWalletAnswer: string]: string;
+  [answerIndex: string]: string;
+}
+
+interface IdentityVerificationAnswersRequest {
+  answers: number[];
+  orgExternalId: string;
+  sessionId: string;
 }
 
 interface IdentityVerificationRequest {
```

Comment:
Should `orgExternalId` be added to the `/get-questions` request now, or separately?

Reply:
Added in d6f19d5!